app/models/ems_cluster.rb

```diff
@@ -246,32 +246,20 @@ def file_shares
     all_hosts.collect(&:file_shares).flatten.uniq
   end
 
-  def event_where_clause(assoc = :ems_events)
-    return ["ems_cluster_id = ?", id] if assoc.to_sym == :policy_events
-
-    cond = ["ems_cluster_id = ?"]
-    cond_params = [id]
+  def event_where_clause(scope, assoc = :ems_events)
+    scope = scope.where("ems_cluster_id" => id)
+    return scope if assoc.to_sym == :policy_events
 
     ids = all_host_ids
-    unless ids.empty?
-      cond << "host_id IN (?) OR dest_host_id IN (?)"
-      cond_params += [ids, ids]
-    end
+    scope = scope.or("host_id" => ids).or("dest_host_id" => ids) unless ids.empty?
```

Comment:
None of these calls are how `or` is supposed to work. [The README](https://github.com/Eric-Guo/where-or#usage) sounds right.. does the gem support this API as well? :confused: